atlp-rwanda/atlp-pulse-mobile
# #1 App onboarding

### What does this PR do?

This PR includes designs for the app onboarding

### Description of Task to be completed?

Design the onboarding page

### How should this be manually tested?

- Run `npm start` to start the metro server.
- Open you device or emulator with expo go app installed and scan the QR code presented in your terminal.
- The onboarding pages should be the first to display

### Screenshots

<img width="220px" src="https://github.com/user-attachments/assets/66d0df29-a245-4d56-a803-f69be2d661a1" />
<img width="220px" src="https://github.com/user-attachments/assets/7e9b8a85-f043-4929-ba05-2b29008e40c5" />
<img width="220px" src="https://github.com/user-attachments/assets/dcf55fea-7ee7-444e-8713-8f0af42a74bd" />
